Dockerfile

```diff
@@ -27,31 +28,13 @@ RUN go build \
     -X github.com/rancher/elemental-toolkit/internal/version.gitCommit=$ELEMENTAL_COMMIT" \
     -o /usr/bin/elemental
 
-FROM opensuse/leap:$LEAP_VERSION AS elemental
+FROM $BUILD_OS_DISTRI:$BUILD_OS_VERSION AS elemental
 # This helps invalidate the cache on each build so the following steps are really run again getting the latest packages
 # versions, as long as the elemental commit has changed
 ARG ELEMENTAL_COMMIT=""
 ENV ELEMENTAL_COMMIT=${ELEMENTAL_COMMIT}
 
-RUN ARCH=$(uname -m); \
```

Comment:
Not really sure about this change, removing the dependencies from the elemental-toolkit container will break all currently documented approaches of building derivatives (using the toolkit container to run commands).

Reply:
This Dockerfile is only to compile the elemental CLI, not one of the provided examples. But yes if the toolkit container is used then it may fails.

I will check the docs and find something else, or use a less minimal images (using BCI images means no `zypper` command).

Reply:
As we discussed I revert this. With the split there is now a `Dockerfile.toolkit` which does this and it works for all architectures.